stacks_queues/queue.py

```diff
@@ -13,49 +13,78 @@ def __init__(self):
         self.store = [None] * INITIAL_QUEUE_SIZE
         self.buffer_size = INITIAL_QUEUE_SIZE
         self.front = -1
-        self.rear = -1
+        self.rear = 0
```

Comment:
There's not really anything wrong with initializing this to 0, but since you reset it to '-1' in `dequeue` below when the queue becomes empty, it's best to stay consistent across the board.
```suggestion
        self.rear = -1
```